packages/core/src/Cardano/util/resolveInputValue.ts

```diff
@@ -0,0 +1,17 @@
+import { TxAlonzo, TxIn, Value } from '../types';
+
+/**
+ * Resolves the value of an input by looking for the matching output in a list of transactions
+ *
+ * @param {TxIn} input input to resolve value for
+ * @param {TxAlonzo[]} transactions list of transactions to find the matching output
+ * @returns {Value | undefined} input value or undefined if not found
+ */
+export const resolveInputValue = (input: TxIn, transactions: TxAlonzo[]) => {
+  let value: Value | undefined;
+  transactions.some((transaction) => {
```

Comment:
This would be cleaner:
```suggestion
  return transactions.some((transaction) => {
```

Reply:
Ah yes, it would have to be `transactions.find`